tasks/functions/functions.js

```diff
@@ -13,34 +14,61 @@ function add(a, b) {
  *    firstName: "John"
  *    lastName: "Dou"
  * }
+ *
  */
-function getFullName(object) {
-
-}
-
+/*var person = {
+    firstName: "John",
+    lastName : "Dou"};
+    */
+   function getFullName(firstName, lastName) {
+    var obj = {
+      firstName: firstName,
+      lastName: lastName
+  };
+    return(obj.firstName + " " + obj.lastName);
```

Comment:
you can omit the brackets and write smth like:
return obj.firstName + " " + obj.lastName;

Reply:
done